config/routes.rb

```diff
@@ -10,6 +10,7 @@
 
   resources :memberships do
     post :approval
+    post :set_moderator
```

Comment:
Not blocking: Whenever we add a `post` to a resourceful route, that is an indicator that we are overloading a controller. There are a few ways we could adjust this:

`put /communities/zinc/moderators/:user_id`  would allow us to maintain a collection of active members, where deleting or putting is adding and removing from that set.